configure.ac

```diff
@@ -64,6 +64,14 @@ AS_IF([test "x$enable_selinux" = "xyes"], [
 	AC_SUBST(HAVE_SELINUX)
 ])
 
+AC_ARG_ENABLE([asan],
+    AS_HELP_STRING([--enable-asan], [enable address sanitizer]))
+AS_IF([test "x$enable_asan" = "xyes"], [
+	EXTRA_CFLAGS="$EXTRA_CFLAGS -fsanitize=address"
+	EXTRA_LDFLAGS="$EXTRA_LDFLAGS -fsanitize=address"
+])
```

Comment:
Suggestion: Move it below the `--enable-analyzer` part, as `--enable-asan` is
also mostly just about compiler flags, while apparmor/selinux are things that
are linked in.

Misc: I think that apparmor/selinux might have made more sense using `--with-`
instead of `--enable-`, but I digress.